components/modal/private/manager.jsx

```diff
@@ -95,6 +95,8 @@ class Manager extends React.Component {
 						onClick={this.closeModal}
 					>
 						<Icon name="close" category="utility" size="small" />
+						{/* This file is an example file and should not be used */}
+						{/* eslint-disable-next-line react/jsx-no-literals */}
```

Comment:
could this file be deleted instead?

Reply:
I'd prefer a separate PR.

Reply:
So we can call it out on the release notes

Reply:
There's a lot of cruft in the library. https://github.com/salesforce/design-system-react/blob/master/components/modal/index.jsx#L226

Thankfully, the library is mostly functional programming, so we haven't had a lot of bugs.

Reply:
@kevinparkerson #2189 added for this issue